ext/pdo_mysql/tests/pdo_mysql_prepare_emulated.phpt

```diff
@@ -100,172 +100,172 @@ $db = MySQLPDOTest::factory();
-        prepex(16, $db, 'SELECT label FROM test WHERE :placeholder > 1',
+        prepex(16, $db, 'SELECT label FROM test_prepare_emulated WHERE :placeholder > 1',
             array(':placeholder' => 'id'));
-        prepex(17, $db, 'SELECT :placeholder FROM test WHERE id > 1',
+        prepex(17, $db, 'SELECT :placeholder FROM test_prepare_emulated WHERE id > 1',
             array(':placeholder' => 'id'));
-        prepex(18, $db, 'SELECT :placeholder FROM test WHERE :placeholder > :placeholder',
+        prepex(18, $db, 'SELECT :placeholder FROM test_prepare_emulated WHERE :placeholder > :placeholder',
             array(':placeholder' => 'test'));
 
         for ($num_params = 2; $num_params < 100; $num_params++) {
             $params = array(':placeholder' => 'a');
             for ($i = 1; $i < $num_params; $i++) {
                 $params[str_repeat('a', $i)] = 'some data';
             }
-            prepex(19, $db, 'SELECT id, label FROM test WHERE label > :placeholder',
+            prepex(19, $db, 'SELECT id, label FROM test_prepare_emulated WHERE label > :placeholder',
                 $params, array('execute' => array('sqlstate' => 'HY093')));
         }
 
-        prepex(20, $db, 'DELETE FROM test');
-        prepex(21, $db, 'INSERT INTO test(id, label) VALUES (1, :placeholder), (2, :placeholder)',
+        prepex(20, $db, 'DELETE FROM test_prepare_emulated');
+        prepex(21, $db, 'INSERT INTO test_prepare_emulated(id, label) VALUES (1, :placeholder), (2, :placeholder)',
             array(':placeholder' => 'row'));
-        $stmt = prepex(22, $db, 'SELECT id, label FROM test');
+        $stmt = prepex(22, $db, 'SELECT id, label FROM test_prepare_emulated');
         var_dump($stmt->fetchAll(PDO::FETCH_ASSOC));
 
-        $stmt = prepex(23, $db, 'SELECT id, label FROM test WHERE :placeholder IS NOT NULL',
+        $stmt = prepex(23, $db, 'SELECT id, label FROM test_prepare_emulated WHERE :placeholder IS NOT NULL',
             array(':placeholder' => 1));
         if (count(($tmp = $stmt->fetchAll(PDO::FETCH_ASSOC))) != 2)
             printf("[024] '1' IS NOT NULL evaluates to true, expecting two rows, got %d rows\n", $tmp);
 
-        $stmt = prepex(25, $db, 'SELECT id, label FROM test WHERE :placeholder IS NULL',
+        $stmt = prepex(25, $db, 'SELECT id, label FROM test_prepare_emulated WHERE :placeholder IS NULL',
             array(':placeholder' => 1));
         if (count(($tmp = $stmt->fetchAll(PDO::FETCH_ASSOC))) != 0)
             printf("[026] '1' IS NOT NULL evaluates to true, expecting zero rows, got %d rows\n", $tmp);
 
-        prepex(27, $db, 'DROP TABLE IF EXISTS test');
-        prepex(28, $db, 'CREATE TABLE test(id INT, label CHAR(255)) ENGINE=MyISAM');
-        if (is_object(prepex(29, $db, 'CREATE FULLTEXT INDEX idx1 ON test(label)'))) {
-            prepex(30, $db, 'INSERT INTO test(id, label) VALUES (1, :placeholder)',
+        prepex(27, $db, 'DROP TABLE IF EXISTS test_prepare_emulated');
```

Comment:
Feels like this is now testing something else? And possibly should be split into another test?